docs/serverless/rules/detections-logsdb-impact.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[detections-logsdb-index-mode-impact]]
+= Using logsDB index mode with {sec-serverless}
+
+LogsDB is enabled by default for Elastic serverless. This topic explains the impact of using logsDB index mode with  {sec-serverless}.  
+
+With logsDB index mode, the original `_source` field is not stored in the index but can be reconstructed using {ref}/mapping-source-field.html#synthetic-source[synthetic `_source`].
+
+When the `_source` is reconstructed, you can make {ref}/mapping-source-field.html#synthetic-source-modifications[modifications] close to the original data. However, there could be a mismatch between how users expect field values to be formatted.
+
+Continue reading to find out how this affects specific {sec-serverless} components. 
+
+[discrete]
+[[logsdb-alerts]]
+== Alerts
+
+When alerts are generated, the `_source` event is copied over into the alert in order to retain the original data. When the logsDB index mode is applied, the `_source` event stored in the alert is an event reconstructed using synthetic `_source`.
+
+If you're switching to use logsDB index mode, the `_source` field stored in the alert might look different in certain situations:
+
+* {ref}/mapping-source-field.html#synthetic-source-modifications-leaf-arrays[Arrays can be reconstructed differently or deduplicated]
+* {ref}/mapping-source-field.html#synthetic-source-modifications-field-names[Field names] 
+
+Alerts generated by the following rule types could be affected:
+
+* Custom query
+* Event correlation (non-sequence only)
+* Non-aggregate rule types
+
+Alerts that are generated by threshold, {ml}, and event correlation sequence rules are not affected since they do not contain copies of the original source.
+
+[discrete]
+[[logsdb-rule-actions]]
+== Rule actions
+
+While we do not recommend using `_source` for actions, in cases where the action relies on the `_source`, the same limitations and changes apply.
+
+If you send alert notifications by enabling {kibana-ref}/alerting-getting-started.html#alerting-concepts-actions[actions], and then have logic outside of {sec-serverless} based on fields formatted from the original source, those flows may be affected. In particular, affected fields would be an array of objects.
```

Comment:
This whole paragraph is a little confusing to me and I'm not 100% sure what it means. Is there any way we could break this down into simpler terms?

Reply:
Maybe something like this: "If you send alert notifications by enabling {kibana-ref}/alerting-getting-started.html#alerting-concepts-actions[actions], and the external systems have workflows or automations based on fields formatted from the original source, they may be affected. In particular, this can happen when the fields used are arrays of objects."

Reply:
```suggestion
If you send alert notifications by enabling {kibana-ref}/alerting-getting-started.html#alerting-concepts-actions[actions] to the external systems that have workflows or automations based on fields formatted from the original source, they may be affected. In particular, this can happen when the fields used are arrays of objects.
```